main.py

```
@@ -0,0 +1,159 @@
+        print(f"    Block #{mined_block_1.index} added!")
+        for tx in mined_block_1.transactions:
+            result = state.apply_transaction(tx)
+            # If a contract was created, we want to know its address
+            if result and isinstance(result, str): 
+                contract_address = result
+                print(f"    -> New Contract Deployed at: {contract_address[:10]}...")
+
+    # ------------------------------------------------------------------
+    # SCENARIO C: Interact with Contract (Bob calls the contract)
+    # ------------------------------------------------------------------
+    print(f"\n[5] Interaction: Bob sends data 'Hello Blockchain' to Contract")
```

Ruff (0.15.0)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)